NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0 Release
```

Comment:
Please follow the NEWS format described in the Best Practices section of [Acquisitions Release Procedures](https://wiki.folio.org/display/FOLIJET/Acquisitions+-+Release+Procedures#Acquisitions-ReleaseProcedures-BestPractices)